### PR TITLE
Moving from swig to ejs

### LINKED ISSUE #156: Move all .swig files to Nunjucks so we can support better rendering
This is a bounty issue for @nanspro to change all our swig files to Nunjucks as swig is no longer maintained, Nunjucks is far more extensible and better, and it will allows us to support critical things like `flow`, `sequence` and `mermaid` MD renderers so that our docs are truly awesome.

## Requirments

Change all swig files to Nunjucks. The site should otherwise look and behave exactly the same.

### PR DESCRIPTION
**Pull request by nanspro**
All .swig files has been changed to .ejs
Fixes #156

### CODE REVIEW

**Comment by nanspro** [comment]
@andytudhope  i have manually checked various routes and it looks fine!!

**Comment by richard-ramos** [comment]
Hola, @nanspro! Since Andy is OoO, I'll do my best to review the PR between today and tomorrow. Thanks for your patience

File: themes/navy/layout/partial/comment.ejs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% if page.comments && config.disqus_shortname %}
+<% if(page.comments && config.disqus_shortname){ %>
 <section id="comments">
   <div id="disqus_thread"></div>
 </section>
```
Comment (richard-ramos) [inline themes/navy/layout/partial/comment.ejs]: On L6-10, Shouldn't these values be changed to the EJS variable tag equivalent?

File: themes/navy/layout/partial/header.ejs [hunk 2]
```diff
@@ -1,13 +1,13 @@
 <div class="header">
     <div class="header-left">
         <a class="logo" href={{url_for_lang('/')}}></a>
```
Comment (richard-ramos) [inline themes/navy/layout/partial/header.ejs]: This appears to be a swig tag

File: themes/navy/layout/partial/header.ejs [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
     </div>
     <div class="header-right">
         <a href="https://get.status.im/chat/public/status" class="button">Join Status Chat</a>
     </div>
 </div>
-
+<%#
 <div class="mobile-nav-wrap">
     <div class="mobile-nav-inner">
         <a class="logo" href={{url_for_lang('/')}}></a>
```
Comment (richard-ramos) [inline themes/navy/layout/partial/header.ejs]: Swig tag?

Comment (richard-ramos) [reply themes/navy/layout/partial/header.ejs]: Also, on Line 17, and 27.

File: themes/navy/layout/partial/header.ejs [hunk 4]
```diff
@@ -35,4 +35,5 @@
 
 {{ partial('partial/shared-partials/partials/community-popup') }}
```
Comment (richard-ramos) [inline themes/navy/layout/partial/header.ejs]: Shouldn't this and the next line be converted to a EJS tag too?

File: themes/navy/layout/privacy-policy.ejs [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <div class="page-intro wrapper">
   <header class="article-header">
     <h1 class="article-title" itemprop="name">{{ page.title }}</h1>
```
Comment (richard-ramos) [inline themes/navy/layout/privacy-policy.ejs]: Convert to EJS tag

**Comment by nanspro** [comment]
@richard-ramos  Done!!

**Comment by richard-ramos** [comment]
Good work, @nanspro

**Pull request merged**